packages/core/src/utilities/frameRangeUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { triggerEvent, eventTarget } from '@cornerstonejs/core';
+
+export type FramesRange = [number, number] | number;
+
+/**
+ * This class handles the annotation frame range values for multiframes.
+ * Mostly used for the Video viewport, it allows references to
+ * a range of frame values.
+ */
+export default class FrameRangeUtils {
```

Comment:
can we name the file at least to FrameRangeUtils so that it is clear it is a class?